migrations/versions/7d6c441557e8_.py

```
@@ -0,0 +1,45 @@
+"""empty message
```

Ruff (0.8.2)

File `migrations/versions/7d6c441557e8_.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)